homeassistant/components/airobot/strings.json

```
@@ -43,6 +43,25 @@
       }
     }
   },
+  "entity": {
+    "sensor": {
+      "air_temperature": {
+        "name": "Air temperature"
+      },
+      "device_uptime": {
+        "name": "Device uptime"
+      },
+      "errors": {
+        "name": "Errors"
```

```suggestion
        "name": "Error count"
```
I think we can make it clearer that this is a numerical value